src/glossary.tex

```diff
@@ -1,3 +1,4 @@
 \makeglossaries{}
 
 \newacronym{epos}{EPOS}{electronic point of sale}
+\newacronym{bi}{BI}{Business Intelligence}
```

Comment:
Should be lowercase, so it can be used in the middle of sentences too. Business intelligence is not a title and should therefore not be capitalized.